src/models/room-state.ts

```diff
@@ -198,8 +199,10 @@ export class RoomState extends TypedEventEmitter<EmittedEvents, EventHandlerMap>
     public constructor(
         public readonly roomId: string,
         private oobMemberFlags = { status: OobStatus.NotStarted },
+        isStartTimelineState: boolean = false,
     ) {
         super();
+        this._isStartTimelineState = isStartTimelineState;
```

Comment:
Why not use the shorthand like the above fields?

Reply:
`private readonly isStartTimelineState = false` then we don't need the getter either